src/styles/home.css

```
@@ -1,785 +1,182 @@
-  align-items: center;
-  gap: 0.65rem;
-  min-width: 0;
-  flex-wrap: wrap;
-  justify-content: flex-end;
-}
-
-.home-theme-switch {
-  display: inline-flex;
-  align-items: center;
-  justify-content: center;
+  font-family: 'Manrope', system-ui, sans-serif;
```

Stylelint (17.13.0)

[error] Expected no quotes around "Manrope" (font-family-name-quotes)

(font-family-name-quotes)